### PR TITLE
[RLlib] Migrate all remaining RLlib release tests to GCE 

### PR DESCRIPTION
**Pull request by ArturNiederfahrenhorst**
## Why are these changes needed?

This PR migrates the remaining RLlib release tests to GCE.
After solving issues outside of RLlib (some still open), I was able to run the already migrated tests:
https://buildkite.com/ray-project/release-tests-branch/builds/1591#_

The only clear regression is with IMPALA.
I suspect that slower sampling together with the asynchronous nature of IMPALA makes for update steps of fewer samples that destabilize training compared to what we see on AWS. Therefore, we'll want to recalibrate the test for GCE.
This is orthogonal to moving forward with putting more release tests onto GCE.

### CODE REVIEW

File: release/release_tests.yaml [hunk 1]
```diff
@@ -3432,29 +3458,16 @@
...
-#    cluster_env: app_config.yaml
-#    cluster_compute: 32cpus.yaml
-
-#  run:
-#    timeout: 18000
-#    script: python learning_tests/run.py --yaml-sub-dir=a3c --framework=torch
-#    type: anyscale_job
-#    file_manager: job
-
-#  alert: default
```
[the remark this answers is not in the document]
Comment (can-anyscale) [reply release/release_tests.yaml:3457]: Nice!

File: release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    instance_type: n2-standard-8 # m5.2xlarge
+
+worker_node_types:
+    - name: worker_node
+      instance_type: n1-standard-4-nvidia-tesla-t4-1 # g4dn.xlarge
+      min_workers: 2
+      max_workers: 2
+      use_spot: false
+
+aws:
```
Comment (can-anyscale) [inline release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml:17]: this block only works for aws, we can remove this, by default gce use 150gb disk

File: release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
```
Comment (can-anyscale) [inline release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml:1]: this block is also for aws as well

**Review by can-anyscale — CHANGES_REQUESTED** [review]
Need to update the compute config a bit, otherwise look great

File: release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml (hunk 3 above)
Comment (can-anyscale) [reply release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml:1]: Totally, I think the config for gce looks something like this


```
cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
region: us-west1
allowed_azs: 
    - us-west1-c
```

**Review by can-anyscale — APPROVED** [review]
verynice

File: release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml (hunk 3 above)
Comment (ArturNiederfahrenhorst) [reply release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml:1]: azs us west1 b or c?

Comment (ArturNiederfahrenhorst) [reply release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml:1]: b was for GPU workloads?

Comment (can-anyscale) [reply release/rllib_tests/multi_node_checkpointing_compute_config_gce.yaml:1]: yes, b sounds good!

**Comment by ArturNiederfahrenhorst** [comment]
All RLlib release tests: https://buildkite.com/ray-project/release-tests-pr/builds/36250#_ (without latest A3C change)

**Comment by ArturNiederfahrenhorst** [comment]
Only a3c: https://buildkite.com/ray-project/release-tests-pr/builds/36332

**Review by gjoliver — APPROVED** [review]

**Pull request merged**